client/src/components/PatientProfile.component.js

```diff
@@ -1,67 +1,77 @@
 import React, {Component} from 'react';
-import Navbar from "./Nbar.component";
 import { Link } from 'react-router-dom';
 import axios from 'axios';
+import CreateSymptoms from './CreateSymptoms.component';
 
-//assigned doctor
-//symptoms delete to be change for update
-const PatientEntry = props => (
-  <tr>
-    <td>{props.patientEntry.patientName}</td>
-    <td>{props.patientEntry.form}</td>
-    <td>{props.patientEntry.additionalNote}</td>
-    <td>{props.patientEntry.doctorNote}</td>
-    <td>{props.patientEntry.immediateAttention}</td>
-    <td>{props.patientEntry.entryDate}</td>
-    <td>
-      <Link to={"/edit/"+props.patientEntry._id}>edit</Link> | <a href="#" onClick={() => { props.deletePatientEntry(props.patientEntry._id) }}>delete</a>
-    </td>
-  </tr>
-)
-
-class PatientProfile extends Component{  
-    constructor(props) {
+export default class PatientProfile extends Component{ 
+   constructor(props){
     super(props);
-        this.state = {patientEntries: []};
+    this.state = {
+        symptom1:'',
+        patientFullName:'',
```

Comment:
In the schema firstName and lastName where a sub array of full name so you might need to do something like this -> patientFullName: {firstName, lastName},

Reply:
Schema was changed to remove patientFullName.